packages/lodestar/src/sync/initial/fast/index.ts

```diff
@@ -135,20 +135,27 @@ export class FastSync
         const setBlockImportTarget = this.setBlockImportTarget;
         const updateBlockImportTarget = this.updateBlockImportTarget;
         const getInitialSyncPeers = this.getInitialSyncPeers;
+        const forkChoice = this.chain.forkChoice;
         return (async function() {
           for await (const slotRange of source) {
             const lastSlot = await pipe(
               [slotRange],
               fetchBlockChunks(
                 logger, chain, network.reqResp, getInitialSyncPeers, opts.blockPerChunk
               ),
-              processSyncBlocks(config, chain, logger, true)
+              processSyncBlocks(config, chain, logger, true, true)
             );
-            logger.verbose("last fetched slot=" + lastSlot);
+            logger.verbose("last processed slot=" + lastSlot);
             if(lastSlot) {
-              //set new target from last block we've received
-              // it will trigger new sync once last block is processed
-              updateBlockImportTarget(lastSlot);
+              if (lastSlot === forkChoice.headBlockSlot()) {
```

Comment:
handle the case when the processSyncBlocks fails at 1st range slot which cause the initial sync suspends